### ISSUE TITLE
broke data:uri

### ISSUE THREAD

**Issue by cssfish** [lavyun/postcss-path-replace#1]
hi! 
nice plugin, thank you. But today it broken data:uri (img) in css.

options:
```
// paths to root
require("postcss-path-replace")({
	publicPath: "../i",
	matched: "../../i",
	mode: "replace"
}),
```
part of input data:uri in css:
```
/9tT9//oTd9/f3/wK8v7+DH/
```

and the same part after plugin:
```
/9tT9//oT../wK8v7+DH/
```

**Comment by cssfish** [comment]
sorry, wrong options. thats right:

// paths to root
require("postcss-path-replace")({
	publicPath: "../",
	matched: "../../",
	mode: "replace"
}),

**Comment by lavyun** [comment]
sorry, can not view your issue.  I fixed this issue in v1.0.4.
